src/screens/Inventory/CreatePurchaseRequest.tsx

```diff
@@ -117,17 +120,17 @@ function CreatePurchaseRequest(props: CreatePurchaseRequestProps) {
             required
             type="number"
             currency
-            label={'Amount Spent'}
+            label={intl(words.amount_spent_paid)}
```

Comment:
Does the `words.amount_spent_paid` correctly translate to `Amount Spent` and not `Amount Spent [Paid]` if no input is given to it?